### ISSUE TITLE
typeText not working in Edge 17

### ISSUE THREAD

**Issue by nzav** [DevExpress/testcafe#2384]
### Are you requesting a feature or reporting a bug?
A bug.

### What is the current behavior?
`typeText` doesn't input values into `input` fields in `Edge 17.17134.0 / Windows 10.0.17134`
Works fine in Chrome/Firefox.

### What is the expected behavior?
It should work :-)

### How would you reproduce the current behavior (if this is a bug)?

1. Create a simple page with an input:
```html
<input type="email" class="uiid-email">
```
2. Try fill it in using `typeText`:
```typescript
const userNameInput = Selector('.uiid-email');
await t.typeText(userNameInput, 'some text');
```

### Specify your

* operating system: Edge 17.17134.0 / Windows 10.0.17134
* testcafe version: 0.19.2
* node.js version: 8.9.4

**Comment by miherlosev** [comment]
Hi @nzav 

Thanks for your bug report.
We've encountered some problems with MSEdge 17 related to typing, focus changing and etc.
It will be fixed ASAP.

**Comment by miherlosev** [comment]
@nzav 

The fix was published in `testcafe@0.20.0-alpha.4`

**Comment by AlFa84** [comment]
I found the same issue with this config:
Edge 17.17134 / Windows 10.0.17134
TestCafè 0.23.1
